src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -47,23 +61,98 @@ public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options)
             }
         }
 
+        private void SetFormatters(IEnumerable<ConsoleFormatter> fromSetup = null)
+        {
+            IEnumerable<ConsoleFormatter> formatters = fromSetup;
+            if (formatters == null || !formatters.Any())
+            {
+                var defaultMonitor = new FormatterOptionsMonitor<SimpleConsoleFormatterOptions>(new SimpleConsoleFormatterOptions());
+                var systemdMonitor = new FormatterOptionsMonitor<ConsoleFormatterOptions>(new ConsoleFormatterOptions());
+                formatters = new List<ConsoleFormatter>()
```

Comment:
>List [](start = 33, length = 4)

I think we can avoid allocating the List here and we can just add the 2 entries directly to the _formatters then exit.